docs/additional-postgres-setup.md

```diff
@@ -1,16 +1,18 @@
 # Setup your application with PostgreSQL
-Follow these links for detailed the installation guide.
 
-1. [Mac OS](https://postgresapp.com/)
-2. Linux / Ubuntu
-   - [Ubuntu `14.04`](https://www.digitalocean.com/community/tutorials/how-to-install-and-use-postgresql-on-ubuntu-14-04)
-   - [Ubuntu `16.04 and higher`](https://www.digitalocean.com/community/tutorials/how-to-install-and-use-postgresql-on-ubuntu-16-04)
-3. [Windows](https://www.postgresql.org/download/windows/)
+Follow the instructions in the the installation guide below that corresponds to your operating system.
```

Comment:
@ocnly, I've updated the wording.

Reply:
That's good, but a duplicate definite article managed to slip in though. Other than that, this sounds much better. Thanks.

Reply:
Done. Got rid of my double-vision. 👓